src/login/login.html

```diff
@@ -0,0 +1,23 @@
+<form>
+  
+    <div class="container">
+      <label for="uname"><b>Email</b></label>
+      <input type="text" placeholder="Enter Email" name="ename" required>

```

Comment:
Could we also add some comments explaining what things do for future students?

Reply:
I added some comments, thanks for the feedback